app.js

```diff
@@ -0,0 +1,28 @@
+const menu = document.querySelector(".menu");
+const menuItems = document.querySelectorAll(".menuItem");
+const hamburger= document.querySelector(".hamburger");
+const closeIcon= document.querySelector(".closeIcon");
+const menuIcon = document.querySelector(".menuIcon");
+
+function toggleMenu() {
+    if (menu.classList.contains("showMenu")) {
+      menu.classList.remove("showMenu");
+      closeIcon.style.display = "none";
+      menuIcon.style.display = "block";
+    } else {
+      menu.classList.add("showMenu");
+      closeIcon.style.display = "block";
+      menuIcon.style.display = "none";
+    }
+  }
+
+  // Clicking on the hamburger button will call toggleMenu()
+hamburger.addEventListener("click", toggleMenu);
+
+//iterate through each link and add a call to toggleMenu().
+menuItems.forEach( 
```

Comment:
Good use of ES6 functionality of the forEach method ⭐ 

Logically understandable